### DISCUSSION TITLE
docker 部署如何开启等级标签

### DISCUSSION THREAD

**Discussion by cirry** [walinejs/waline#1357] [Q&A] [answered]
在docker-compose中environment里设置` LEVELS: "0,10,20,50,100,200" `控制台提示报错如下：

Fetch comment list failed with 500: 字段 "COUNT(*) as count" 不存在

waline | [2022-09-05T12:17:37.591] [14] [INFO] - SQL: SELECT "user_id","mail","COUNT(*) as count" FROM "wl_comment" WHERE ( "status" NOT IN (E'waiting',E'spam') ) AND (  ( "user_id" = 1 ) OR ( "mail" IN (E'cirry',E'8xxxxxxxxq.com',E'xxxxx@yun.com') ) ) GROUP BY "user_id", "mail", Time: 3ms
waline | error: 字段 "COUNT(*) as count" 不存在

**Comment by lizheming** [comment, marked as answer]
`@waline/vercel@1.21.0` 已修复，可以再试试
